code/game/objects/gamemode/crash/machines.dm

```diff
@@ -55,12 +55,8 @@
 	visible_message("<b>[src]</b> shuts down as it loses power. Any running programs will now exit")
 
 
-/obj/machinery/computer/nuke_disc_generator/attackby(obj/item/I, mob/user, params)
+/obj/machinery/computer/nuke_disc_generator/attackby(obj/item/I, mob/living/user, params)
 	. = ..()
```

Comment:
Also depending on what you want, this will allow a bunch of extra interactions. If you want to just open the interface regardless of anything, then you may want to remove this call.